src/styles/main.scss

```diff
@@ -1,7 +1,727 @@
+      margin: 0 auto;
+
+      &__title {
+        font-size: 48px;
+        font-weight: 700;
+        text-align: center;
+        letter-spacing: -1.5px;
+
+        @include changeSize($mobile) {
+          width: 280px;
+          font-size: 32px;
+          letter-spacing: -1.5px;
+        }
+      }
+    }
+
+    .section__block {
+      margin-top: 48px;
+      display: grid;
+      grid-template-columns: repeat(3, 1fr);
+      column-gap: 24px;
+      justify-content: center;
+
+      @include changeSize($mobile) {
+        grid-template-columns: 1fr;
+      }
+
+      .section__col-4 {
+
+        @include changeSize($mobile) {
+          padding: 20px 20px;
+          width: 280px;
+        }
+
+        .section__photos {
+          width: 324px;
+          height: 320px;
+          transition: transform 0.3s ease-in;
+
+          &:hover {
+            cursor: pointer;
+            transform: scale(1.1);
+          }
+
+          @include changeSize($mobile) {
+            width: 288px;
+            height: 287px;
+          }
+        }
+
+        .title__area {
+          font-size: 24px;
+          color: #d12d35;
+          font-weight: 700;
+          text-align: center;
+          margin-bottom: 10px;
+          box-sizing: border-box;
+        }
+
+        .title__area-info {
+          font-family: "Roboto", sans-serif;
+          font-weight: 400;
+          font-size: 16px;
+          text-transform: none;
+          text-align: center;
+          color: #161616;
+          font-variant: normal;
+        }
+      }
+    }
+  }
+
+  &__section-4 {
+    .contact {
+      text-align: center;
+      &__title {
+        font-size: 48px;
+        font-weight: 700;
+        color: #161616;
+        margin-top: 140px;
+      }
+    }
+    .section__block {
+      display: grid;
+      grid-template-columns: repeat(3, 1fr);
+      column-gap: 70px;
+      justify-content: center;
+
+      @include changeSize($mobile) {
+        grid-template-columns: 1fr;
+      }
+
+      .col-1 {
+
+        @include changeSize($mobile) {
+          padding: 20px 20px;
+        }
+
+        &__title {
+          font-weight: 700;
+          font-size: 16px;
+          color: #161616;
+        }
+
+        &__phone,
+        &__address-text {
+          font-weight: 400;
+          font-size: 14px;
+          color: #161616;
+        }
+
+        &__number {
+          font-weight: 700;
+          font-size: 16px;
+          color: #3e3e3e;
+          text-decoration: none;
+        }
+
+        &__address {
+          font-weight: 700;
+          font-size: 16px;
+          color: #3e3e3e;
+          text-transform: uppercase;
+          text-decoration: none;
+        }
+
+        .contact__name,
+        .contact__email,
+        .contact__message,
+        .contact__send-button {
+          display: block;
+          width: 410px;
+          margin-top: 27px;
+          border-color: transparent;
+          box-sizing: border-box;
+        }
```

Comment:
we've discussed with you this whole styles nesting grotesque...
if you're nesting, then you use `&...`
```
        .contact {
          &__name,
          &__email,
          &_message {
            ...styleset
          }
        }
```

if style is not a descendant of the nest parent, you don't nest it there
```
.parent-class {
  &__dependent-class {
    ...styleset
  }

  &:<pseudo-element / class / modifier> {
    ...styleset
  }
}

.not-dependent-style {
 ...styleset
}
```